ocaml/doc/pdf/wire-protocol.md

```diff
@@ -0,0 +1,316 @@
+# Wire Protocol for Remote API Calls
+
+API calls are sent over a network to a Xen-enabled host using
+the [XML-RPC](http://xmlrpc.scripting.com/spec.html) protocol. Here we describe
+how the higher-level types used in our API Reference are mapped to primitive
+XML-RPC types.
+
+We specify the signatures of API functions in the following style:
+
+```python
```

Comment:
This (and similar below) isn't Python. Are you just using it to make things look nice in the pandoc output?